### ISSUE TITLE
Floating point parsing error in Firefox

### ISSUE THREAD

**Issue by jalbarl** [n3r4zzurr0/svg-spinners#1]
Most of the animations don't work on Firefox (102 beta 6).
The error in the log for [this example](https://raw.githubusercontent.com/n3r4zzurr0/svg-spinners/main/svg/270-ring-with-bg.svg) is:

> Unexpected value .75s parsing dur attribute.

Apparently Firefox does not support decimal numbers without a leading digit, which might be standard-conforming and not a bug in Firefox. Fixing the number to read 0.75 immediately restores the animation for the example above.
I looked for a bit into the SVG specification and I found a [statement](https://svgwg.org/specs/animations/#ClockValueSyntax) related to the "dur" attribute format in this particular case:

`Timecount-val     ::= Timecount ("." Fraction)? (Metric)?`

In other words, the decimal point, the fractional part and the metric are optional, but the first digit must always be present.

**Comment by n3r4zzurr0** [comment]
All the `dur`, `begin` and `end` parameters that started with a leading decimal point have been updated with a leading `0` as per [SMIL specifications](https://www.w3.org/TR/SMIL3/smil-timing.html#q7).
